Lib/Protocols/IdFTPServer.pas

```diff
@@ -3948,6 +3995,9 @@ procedure TIdFTPServer.CommandFEAT(ASender: TIdCommand);
   if (UseTLS <> utNoTLSSupport) and (LContext.Binding.IPVersion = Id_IPv4) then begin
     ASender.Reply.Text.Add('CPSV');   {Do not translate}
   end;
+  if Assigned(FOnClientID) then begin
+    ASender.Reply.Text.Add('CSID');  {Do not localize}

```

Comment:
Since a `CSID` reply is sent back to the client regardless of whether the `OnClientID` event is assigned, this `FEAT` entry should not be conditional.